### PR TITLE
codacy issues set1

### PR DESCRIPTION
**Pull request by aayusharora**
@juslee ,@championswimmer , this is in regard of [this](https://www.codacy.com/app/triplez/open-event-webapp/issues?bid=3233857&step=20&filters=W3siaWQiOiJMYW5ndWFnZSIsInZhbHVlcyI6W251bGxdfSx7ImlkIjoiQ2F0ZWdvcnkiLCJ2YWx1ZXMiOlsiRXJyb3IgUHJvbmUiXX0seyJpZCI6IkxldmVsIiwidmFsdWVzIjpbbnVsbF19LHsiaWQiOiJQYXR0ZXJuIiwidmFsdWVzIjpbbnVsbF19XQ%3D%3D)

### CODE REVIEW

File: src/appComponents/twitterwall/singletweet.js [hunk 1]
```diff
@@ -44,20 +44,12 @@ function tweet($timeout,$rootScope) {
...
-        });
-        function getObject(){
-            scope.$watch('array',function(oldvalue,newvalue){
-            if(scope.array){
-            scope.selectedid=scope.array[1].id_str;
-            changeTweet();
-            }
-            })
+         
+        function callchangetweet(){
```
Comment (juslee) [inline src/appComponents/twitterwall/singletweet.js]: Codacy Issue found: ["changeTweet" was used before it was defined](https://www.codacy.com/app/triplez/open-event-webapp/file/2408814751/issues/source?bid=3233857&fileBranchId=3234618#l49)

Comment (aayusharora) [inline src/appComponents/twitterwall/singletweet.js]: @juslee ,@championswimmer , this is against the code structure i follow , "[sir john papa style](https://github.com/johnpapa/angular-styleguide) " , Should I ignore it.

File: src/appComponents/twitterwall/singletweet.js [hunk 2]
```diff
@@ -68,8 +60,19 @@ function tweet($timeout,$rootScope) {
                 callchangetweet();
             }
             }
-        function callchangetweet(){
-              $timeout(changeTweet,8000);
-        }   
+       
+        function getObject(){
```
Comment (juslee) [inline src/appComponents/twitterwall/singletweet.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2408883738/issues/source?bid=3233857&fileBranchId=3234618#l65)

File: src/appComponents/twitterwall/singletweet.js (hunk 1 above)
Comment (juslee) [inline src/appComponents/twitterwall/singletweet.js]: Codacy Issue found: ["changeTweet" was used before it was defined](https://www.codacy.com/app/triplez/open-event-webapp/file/2408883738/issues/source?bid=3233857&fileBranchId=3234618#l49)

**Pull request merged**